### DISCUSSION TITLE
米游社推送优化

### DISCUSSION THREAD

**Discussion by BTMuli** [Arondight/Adachi-BOT#911] [Ideas]
能否获取推送咨讯的时候加一个条件，用以验证咨讯发送日期跟系统当前日期，然后将早于系统日期的推送给忽略掉，
以防止停止应用后过了一阵子重新启动时的过时咨讯轰炸？

**Comment by mark9804** [comment]
> 将早于系统日期的推送给忽略掉

当时这个功能有点问题（好吧主要是我网络的问题）会导致特定情况下不推送，所以推送早于系统日期的消息某种意义上是个 feature

你可以暂时在 `data/db/news.json` 里面修改 `timestamp` 的值来完成不推送的需求

**Comment by Arondight** [comment]
> 能否获取推送咨讯的时候加一个条件

可以加
